stories/Colors.mdx

```diff
@@ -8,42 +10,53 @@ import { colors, default as aliasColors } from '@/colors.ts';
 Base colours provide a full colour spectrum of dark to light hues, and are used as aliases in the [brand theme](#brand-theme-and-ui-colours).
 
 ## Base Colours
-#
 
 <ColorPalette>
-
   {Object.keys(colors).map((token, index) => {
-    const color = colors[token];
+    const color = colors[token]
     return (
-      <ColorItem
-        key={index}
-        index={index}
-        title={`${token}`}
-        colors={color}
-      />
-    );
+      <ColorItem key={index} index={index} title={`${token}`} colors={color} />
+    )
   })}
-
 </ColorPalette>
+## Brand Theme and UI Colours ### Accent
 
-## Brand Theme and UI Colours
-#
+<ColorWrapper colorPalette={aliasColors.accent} />
 
-<ColorPalette>
+### Feedback
 
-  {Object.keys(aliasColors).map((token, index) => {
-    const color = aliasColors[token];
-    if (typeof color === 'object' && !colors[token]) {
-      return (
-        <ColorItem
-          key={index}
-          index={index}
-          title={`${token}`}
-          colors={color}
-        />
-      )
-    }
-  })}
+<ColorPalette>
+{<ColorItem
+  key={0}
+  index={0}
+  title={'feedback'}
+  colors={aliasColors.feedback}
+/>}
 
 </ColorPalette>
+### Background
+<ColorWrapper colorPalette={aliasColors.background} />
+
+### Foreground
+
+<ColorWrapper colorPalette={aliasColors.foreground} />
+### Border
+
+<ColorWrapper colorPalette={aliasColors.border} />
+## Button
+
+### Primary
+
+<ColorWrapper colorPalette={aliasColors.button.primary} />
+
+### Secondary
 
+<ColorWrapper colorPalette={aliasColors.button.secondary} />
+### Secondary Borders
+<ColorWrapper colorPalette={aliasColors.button.secondary.border} />
+### Tertiary
+<ColorWrapper colorPalette={aliasColors.button.tertiary} />
+### Success
+<ColorWrapper colorPalette={aliasColors.button.success} />
+### Danger
+<ColorWrapper colorPalette={aliasColors.button.danger} />
```

Comment:
It looks like the `ColorWrapper` component isn't working for these three colour sets, it's rendering a blank colour swatch for `dark` where it isn't defined. 

Can we also simplify these success and danger colours and just have a single row for each?